ui.apps/src/main/content/jcr_root/apps/aem-bootcamp/components/structure/breadcrumb/breadcrumb.html

```diff
@@ -0,0 +1,20 @@
+<sly data-sly-use.breadcrumb="com.aembootcamp.core.models.BreadcrumbModel" />
+<sly data-sly-test="${!breadcrumb.isHomePath}">
```

Comment:
change this to breadcrumb.homePage

reference: https://stackoverflow.com/questions/67930228/what-is-the-difference-between-using-is-or-get-in-java-when-calling-it-with